src/applications/disability-benefits/686c-674/config/chapters/report-add-a-spouse/current-spouse-marriage-history-details/currentSpouseMarriageHistoryDetails.js

```diff
@@ -1,7 +1,14 @@
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
 import { addSpouse } from '../../../utilities';
 import { SpouseTitle } from '../../../../components/ArrayPageItemSpouseTitle';
-import { stateTitle, cityTitle } from '../../../helpers';
+import {
+  stateTitle,
+  cityTitle,
+  isChapterFieldRequired,
```

Comment:
I see this being added as an import but it doesn't seem to be used in the diff. Does this need to be here?

Reply:
good catch, removed